src/components/ModelNavigator/DataDictionary/Table/DataDictionaryPropertyTable/component/CDEListComponent.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import {
+    List,
+    ListItem,
+    ListItemText,
+    ListItemIcon,
+    withStyles,
+    Typography,
+    createTheme,
+    MuiThemeProvider,
+} from '@material-ui/core';
+import FiberManualRecord from '@material-ui/icons/FiberManualRecord';
+import {
+    addHighlightingSpans,
+} from '../../../Utils/highlightHelper';
+
+const linkPattern = /\b(?:https?|ftp):\/\/(?:www\.)?[^\s/$.?#].[^\s]*\b/;
+function isLink(str) {
+    return linkPattern.test(str);
+}
+
+const wrapLinkInLink = (inputString) => {
+    const match = inputString.match(linkPattern); 3
+
+    if (match) {
+        const link = match[0];
+        const linkIndex = inputString.indexOf(link);
+
+        return (
+            <>
+                {inputString.substring(0, linkIndex)}
+                <a href={`mailto:${link}`}>{link}</a>
```

Comment:
This probably shouldn't be a `mailto:...` link, right? It's opening the URL in Outlook.

![Screenshot 2024-10-15 at 1 00 15 PM](https://github.com/user-attachments/assets/1c320984-8c75-4191-82f0-1d6bcb8bedf0)